src/netstandard/src/netstandard.csproj

```diff
@@ -24,13 +25,23 @@
     <ComputeNETCoreBuildOutputFiles>false</ComputeNETCoreBuildOutputFiles>
     <GenerateRuntimeConfigurationFiles>false</GenerateRuntimeConfigurationFiles>
   </PropertyGroup>
+  
+  <ItemGroup>
+      <!-- Omit System.Void since C# doesn't permit the use of the `System.Void`
+         typename and instead requires callers to use the `void` keyword. -->
+    <GenFacadesOmitType Include="System.Void" />
```

Comment:
We may not require this if we are already setting ```GenFacadesIgnoreMissingTypes``` to true

Reply:
This isn't a missing type.  It's a type that has a different representation for `typeof` than any other type.  When we need to use it we just tell genfacades not to write it and we write it ourselves.  We're doing this because I suggested it here: https://github.com/dotnet/arcade/pull/2221#discussion_r268312871.  We could add back the code to GenFacades to handle this and remove the workaround.  My main motivation in the suggestion before was to ensure we had an escape hatch in case we needed to suppress types.